react/features/base/styles/shimStyles.web.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Shim style properties to work correctly on Web.
```

Comment:
We haven't reached an agreement of how to do styles in React, so let's not go there yet.